packages/create-sitecore-jss/src/templates/nextjs-sxa/scripts/bootstrap/plugins/compile-styles.ts

```diff
@@ -4,7 +4,7 @@ import sass from 'sass';
 import { minify } from 'csso';
 import { pathToFileURL } from 'url';
 import { getItems } from 'scripts/utils';
-import { projectRootPath } from './../../generate-component-factory';
+import { projectRootPath } from './../../generate-component-factory-creator';
```

Comment:
See previous comment. This seems strange, for example. Instead of `scripts/generate-component-factory-creator.ts` owning the project root path, would this be better owned by `sitecore-jss-dev-tools` (e.g. as a projects sub-module or utility)?

Reply:
Same, let's consider in follow-up item.